src/test/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/PopulateUpdateSubmissionTest.java

```diff
@@ -0,0 +1,190 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.BeneficialOwnerAllFieldsMock;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.DueDiligenceMock;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.EntityMock;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.EntityNameMock;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.ManagingOfficerMock;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.PresenterMock;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.TrustMock;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.UpdateMock;
+import uk.gov.companieshouse.overseasentitiesapi.model.BeneficialOwnersStatementType;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.BeneficialOwnerCorporateDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.BeneficialOwnerGovernmentOrPublicAuthorityDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.BeneficialOwnerIndividualDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.DueDiligenceDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.EntityDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.EntityNameDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.ManagingOfficerCorporateDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.ManagingOfficerIndividualDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.PresenterDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.UpdateDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.trust.TrustDataDto;
+import uk.gov.companieshouse.overseasentitiesapi.service.update.PopulateUpdateSubmission;
+import uk.gov.companieshouse.overseasentitiesapi.validation.*;
+
+@ExtendWith(MockitoExtension.class)
+class PopulateUpdateSubmissionTest {
+
+  private final EntityNameDto entityNameDto = EntityNameMock.getEntityNameDto();
+  private final EntityDto entityDto = EntityMock.getEntityDto();
+  private final PresenterDto presenterDto = PresenterMock.getPresenterDto();
+  private final BeneficialOwnersStatementType beneficialOwnersStatement =
+      BeneficialOwnersStatementType.NONE_IDENTIFIED;
+  private final DueDiligenceDto dueDiligenceDto = DueDiligenceMock.getDueDiligenceDto();
+  private final UpdateDto updateDto = UpdateMock.getUpdateDto();
+  private final List<BeneficialOwnerIndividualDto> beneficialOwnerIndividualDtoList =
```

Comment:
Since these mock initialisations (lines 47-57 and lines 62-87)  are only used once by the `buildOverseasEntitySubmissionDto` method, we could move these into the function